NEWS.txt

```diff
@@ -3,243 +3,29 @@ Release Notes
 =============
 
 
-Theano 1.0.0 (15th of November, 2017)
-=====================================
+Theano 1.0.1 (1st of December, 2017)
+====================================
 
-This is a final release of Theano, version ``1.0.0``, with a lot of
-new features, interface changes, improvements and bug fixes.
+This is a release of Theano, version ``1.0.1``, with important bug fixes.
```

Comment:
I would say this is a maintenance release of Theano, with no new features, but some important bug fixes.